app/controllers/api/v1/classifications_controller.rb

```diff
@@ -18,6 +20,13 @@ def update
 
   private
 
+  def filter_by_subject_id
+    if subject_ids = params.delete(:subject_id).try(:split, ',')
```

Comment:
Should we test if the returned array from split is empty instead of just truthy here? I.e a `{ params[:subject_id] => "" }` would result in an empty array for subject_ids due to `p "test" if "".split(",")`.